modules/travisci/client.go

```diff
@@ -39,22 +42,25 @@ var (
 
 func travisBuildRequest(settings *Settings) (*http.Response, error) {
 	var path string = "builds"
+	if settings.baseURL != "" {
+		travisAPIURL.Path = "/api/"
+	}
```

Comment:
I would consider moving these to the settings file and simplifying this here to just be whatever settings exist.

Not a requirement and thanks for the contribution!